File: Java/프로그래머스/0/120884. 치킨 쿠폰/치킨 쿠폰.java
class Solution {
    public int solution(int chicken) {
        int answer = 0;
        int coupoun=chicken;
        int sub=0;
        while(true){
            if(coupoun<10) break;
            sub+=coupoun/10;
            coupoun=coupoun%10+coupoun/10;
        } 
        answer=sub;
        return answer;
    }
}

File: 프로그래머스/0/120884. 치킨 쿠폰/치킨 쿠폰.js
function solution(chicken) {
    let answer = 0;
    let coupoun=chicken;
    while(true){
        if(coupoun<10) break;
        answer+=Math.floor(coupoun/10);
        coupoun=coupoun%10+Math.floor(coupoun/10);
    } 
    return answer;
}